terraform/aws/minikube/main.tf

```diff
@@ -25,7 +31,7 @@ module "minikube" {
   cluster_name = var.cluster_name
   aws_instance_type = var.aws_instance_type
   aws_region = var.region
-  aws_subnet_id = aws_default_subnet.default.id 
+  aws_subnet_id = var.vpc_id != "" ? tolist(data.aws_subnet_ids.vpc_subnets[0].ids)[0] : aws_default_subnet.default.id
```

Comment:
```terraform
variable "vpc_id" {
  type=string
  description="VPC ID (In case it differs from default)"
  default = ""
}
```

By default, vpc_id - empty, and this part of condition  works fine
But when anyone set custom `vpc_id`, why it always must be 1-st in subnets? Maybe we also should add possibility to specify `subnet-id`, because in vpc can be subnets w/o access to external (internet) resources and cluster creation will failed

Reply:
It would be better to do this as a separate task.

Reply:
@MaxymVlasov please spawn a separate task on subnet definition, and lets merge it.

Reply:
Done
https://github.com/shalb/cluster.dev/issues/14